### ISSUE TITLE
BUG: Builder translates randint to inclusive boundaries

### ISSUE THREAD

**Issue by kdoelling1919** [psychopy/psychopy#2849]
I used builder to create an online experiment. In one custom code component I generate a random integer from 1 to 10. For this I put in the following code:
```python
num = randint(1, 11)
```
Python doesn't include the upper limit so it returns only numbers 1 through 10. 

Builder automatically translates this to the following:
```javascript
num = (Math.floor((Math.random() * ((11 - 1) + 1))) + 1);
```
This will occasionally return 11 and created an error for me when it did. I think removing the + 1 inside the inner parentheses will fix this issue.

My suggestion would be to adjust translation such that
 **python:** `num = randint(a, b)` --> **javascript:** `num = (Math.floor(Math.random() * (b - a)) + a);`

**Comment by kdoelling1919** [comment]
This is somewhat related to #2847 but the problems brought up there are different so I decided to make a new issue.

**Comment by peircej** [comment]
Good catch Keith, I think that makes sense

**Comment by kdoelling1919** [comment]
Great! I made a pull request to fix the issue in #2853

**Comment by kdoelling1919** [comment]
Closing as the merged pull request should fix this.
